### DISCUSSION TITLE
有一处错误，谁来修改下

### DISCUSSION THREAD

**Discussion by masami1912** [WeNeedHome/SummaryOfLoanSuspension#525] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32126457/179163606-40cd76aa-a72a-414e-b87b-1ed106d0b32b.png)
镇江 ：华泰禾金尊府。
句容：泰禾金尊府。
这两处指的是一个楼盘，楼盘位于句容市宝华镇，句容是县级市隶属于镇江市。不会用github，麻烦会的更正下

**Comment by hktk1024** [comment, marked as answer]
https://github.com/WeNeedHome/SummaryOfLoanSuspension/pull/532

帮你提交了

**Reply by CHENQOK** [reply]
289我也提过了
